st3/lsp_utils/__init__.py

```diff
@@ -1,2 +1,16 @@
-from .npm_client_handler import NpmClientHandler
+try:
+    from LSP.plugin import __version__ as lsp_version
```

Comment:
VERSION is used by the release script.
Release script works in two stages - build and publish.
The build stage creates the version file and the publish step uses it to know which version to publish.

We could in theory switch to using only the version.py but it might get a bit ugly.